src/components/topNav/primaryTopNav/PrimaryTopNav.module.css

```diff
@@ -9,6 +9,8 @@
   --conduction-primary-top-nav-toggle-icon-size: 24px;
   --conduction-primary-top-nav-mobile-logo-padding: 18px;
   --conduction-primary-top-nav-item-icon-margin: var(--skeleton-size-2x);
+  --conduction-primary-top-nav-dropdown-background-color: var(--web-app-color-body-background);
```

Comment:
The `--web-app` tokens should either be `--skeleton` tokens or a hard coded CSS value since the `--web-app` token does not exist in this context.